site/content/en/references/kustomize/kustomization/bases/_index.md

```diff
@@ -8,7 +8,11 @@ description: >
 ---
 
 {{% pageinfo color="warning" %}}
-The `bases` field was deprecated in v2.1.0
+The `bases` field was deprecated in v2.1.0. Though this field will never be
+removed from the current Kustomization v1beta1 API, the field will not be included
+in the API for Kustomization v1. When Kustomization v1 is available, we will announce
+the deprecation of Kustomization v1beta1. There will be at least two releases between
+deprecation and removal of Kustomization v1beta1.
```

Comment:
nit: maybe also clarify that the v1beta1 removal itself will happen as part of a future major version bump? 

Here's what we went with in the release notes: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.0.0

Reply:
Will update it!

Reply:
Done.